### ISSUE TITLE
when typescript selected, cypress is not configured for typescript

### ISSUE THREAD

**Issue by xenoterracide** [aurelia/new#25]
https://docs.cypress.io/guides/tooling/typescript-support.html#Configure-tsconfig-json

also if you choose the app with routing, it'd be nice to have at least the most basic test in place

**Comment by 3cp** [comment]
Router test setup was blocked by #15 (a bug in Aurelia 2).

**Comment by 3cp** [comment]
The cypress setup here uses babel for ts project, only for simplicity. I was lazy to properly switch to ts.

It will be updated to babel for babel, and ts for ts, just like aurelia-cli did.
